app/webpacker/components/RegistrationsV2/Register/StepPanel.jsx

```diff
@@ -164,6 +193,9 @@ export default function StepPanel({
             const registrationOverviewIndex = steps.findIndex(
               (step) => step === registrationOverviewConfig,
             );
+            if (overwrites.toEnd) {
```

Comment:
Nitpick: Even though you say "über**schreiben**" in German, it's actually spelled "override" in English.